src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -59,6 +59,8 @@
     protected $startTime;
     protected $loadClassCache;
 
+    private $projectDir;
```

Comment:
Well the issue is that the property is set in `__construct` and later used by `getKernelParameters`. My kernel does not know the path at construct time, so I would need to override multiple methods for no reason…